pallets/file-system/src/mock.rs

```diff
@@ -335,29 +333,25 @@ impl crate::Config for Test {
     type Providers = Providers;
     type ProofDealer = ProofsDealer;
     type Fingerprint = H256;
-    type StorageRequestBspsRequiredType = u32;
+    type ReplicationTargetType = u32;
     type ThresholdType = ThresholdType;
-    type ThresholdTypeToBlockNumber = SaturatingThresholdTypeToBlockNumberConverter;
-    type BlockNumberToThresholdType = BlockNumberToThresholdTypeConverter;
+    type ThresholdTypeToBlockNumber = ThresholdTypeToBlockNumberConverter;
     type HashToThresholdType = HashToThresholdTypeConverter;
     type MerkleHashToRandomnessOutput = MerkleHashToRandomnessOutputConverter;
     type ChunkIdToMerkleHash = ChunkIdToMerkleHashConverter;
     type Currency = Balances;
     type Nfts = Nfts;
     type CollectionInspector = BucketNfts;
-    type AssignmentThresholdDecayFactor = ThresholdAsymptoticDecayFactor;
-    type AssignmentThresholdAsymptote = ThresholdAsymptote;
-    type AssignmentThresholdMultiplier = ThresholdMultiplier;
-    type TargetBspsRequired = ConstU32<3>;
-    type MaxBspsPerStorageRequest = ConstU32<5>;
+    type MaxBspsPerStorageRequest = ConstU32<10>;
     type MaxBatchConfirmStorageRequests = ConstU32<10>;
+    // TODO: this should probably be a multiplier of the number of maximum multiaddresses per storage provider
```

Comment:
Why?

Reply:
If you make any changes here as a consequence of this comment, remember to make them in all the other `mock.rs` files.